cmd/jag/commands/device.go

```diff
@@ -50,16 +50,15 @@ const (
 	pingTimeout = 400 * time.Millisecond
 )
 
-func (d Device) Ping(ctx context.Context) bool {
-	info := GetInfo(ctx)
+func (d Device) Ping(ctx context.Context, sdk *SDK) bool {
```

Comment:
I guess the sdk parameter isn't strictly necessary, if we run `validate` consistently and early enough, as we would already bail when the SDK and the Jag version weren't the same.
Correct?

Reply:
No, that's the whole problem. We skip validation if jaguar is pulling the SDK location from an environment variable (which is super useful when developing), but then we need to pass the right SDK version (the found one, not the configured one) to the device.